addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/AbstractNetatmoThingHandler.java

```diff
@@ -55,6 +57,8 @@ public void initialize() {
         batteryHelper = thing.getProperties().containsKey(PROPERTY_BATTERY_LEVELS)
                 ? Optional.of(new BatteryHelper(thing.getProperties().get(PROPERTY_BATTERY_LEVELS)))
                 : Optional.empty();
+        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED,
```

Comment:
Just asking myself if BRIDGE_UNINITIALIZED is appropriate ?

Reply:
Changed to NONE